test/reader/StorageReadable.t.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
We are still discussing which license we want to use so maybe we'll have to ask you to change it once we settle (likely choices are some combination of LGPL, Apache, and MIT).
Would this work for you?